hbase-common/src/main/java/org/apache/hadoop/hbase/io/FileChangeWatcher.java

```diff
@@ -187,21 +188,18 @@ private class WatcherThread extends Thread {
 
     private static final String THREAD_NAME_PREFIX = "FileChangeWatcher-";
 
-    final WatchService watchService;
-    final Consumer<WatchEvent<?>> callback;
+    final FileChangeWatcherCallback callback;
 
-    WatcherThread(String threadNameSuffix, WatchService watchService,
-      Consumer<WatchEvent<?>> callback) {
+    WatcherThread(String threadNameSuffix, FileChangeWatcherCallback callback) {
       super(THREAD_NAME_PREFIX + threadNameSuffix);
-      this.watchService = watchService;
       this.callback = callback;
       setUncaughtExceptionHandler(FileChangeWatcher::handleException);
     }
 
     @Override
     public void run() {
       try {
-        LOG.info("{} thread started", getName());
```

Comment:
This log line and the corresponding "thread finished" one tend to get logged very frequently and aren't high-value. Lowered priority to DEBUG.